docs/content/getting-started/concepts.md

```diff
@@ -46,6 +46,19 @@ A subset of `Data Samples` selected using a sampling rule usually based on the c
 
 You can take a look at this [section](../learn/samplers.md#configuration) to learn more about how to create `Streams` and this [reference](../reference/rules.md) page to see their syntax.
 
+#### Keyed stream
+
+When a stream is keyed, each `Sample` comes with a key that is used to segment the samples. From the stream point of view, being keyed or not does not change the functionality apart from annotating the generated data with the key. But it changes the functionality of the computations that are perfromed with the data in the following way:
```

Comment:
The definition seems a bit hard to understand. We can get inspiration from the Flink docs and say something like:

```When a stream is keyed, each `Sample` is expected to come with an associated key. This key is used to logically divide the stream into disjoint partitions where all `Samples` with the same key are assigned to the same partition. It is then possible to perform some computations in each partition independently``` 

And then explain the event stateful states as an example and not as an exhaustive list, otherwise we will have to keep updating this part every time we add a computation that makes use of keyed streams.

Reply:
Will do that, but I will skip the part about partitioning. We don't have partitions.

Reply:
Well, after thinking about it. We can keep it as `partition` in this definition is like `set`